jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.py

```diff
@@ -239,7 +239,15 @@ def calculate_moment(self, add_data=True):
         # Need transpose to align JWST mirror shape: This is because specutils
         # arrange the array shape to be (nx, ny, nz) but 2D visualization
         # assumes (ny, nx) as per row-major convention.
-        data_wcs = getattr(cube.wcs, 'celestial', None)
+
+        # Extract 2D WCS from input cube.
+        data = self.dataset.selected_dc_item
```

Comment:
I think this might also fix Patrick's bug where WCS is missing when continuum is used but I cannot find that ticket. Someone please confirm.

Reply:
this might be nice to have in its own PR and backport, no?

Reply:
If someone can find Patrick's JIRA ticket, I can do a manual backport later with just this part. Kinda painful to separate it out now.

Reply:
I don't see a ticket, but think its worth fixing as a bugfix since you've already done the work 🤷‍♂️

Reply:
Did I imagine that Patrick filed a ticket? 🤪 

Okay, I can start on the direct backport now. Should be simple.

Reply:
Actually, @kecnry , I don't think a backport is needed because turns out https://github.com/spacetelescope/jdaviz/pull/2587 is milestoned for 3.9 and this bug only affects continuum stuff.

Reply:
ah ok, great! 🎉